backend/src/controllers/rentals.py

```diff
@@ -1,22 +1,31 @@
-from flask import Blueprint, abort
+from flask import Blueprint, Response, abort, flash, jsonify, request, session
 from flask_login import login_required, current_user
 
+from ..config.extensions import bcrypt, db, login_manager
 from ..models.rental import Rental
 from ..models.user import User
 from ..models.vehicle import Vehicle
 from ..models.insurance import Insurance
+import base64
 
-rental = Blueprint('rental', __name__, url_prefix='/rental')
+rentals = Blueprint('rentals', __name__, url_prefix='/rentals')
 response_class = Blueprint('response_class', __name__)
 
-
-
-@rental.route('/', methods=['GET'])
+@rentals.route('/', methods=['GET'])
 @login_required
 def get_all():
     user_permissions = current_user.permissions
     if user_permissions in ["manager", "worker"]:
         rentals = Rental.query.join(User, Rental.client_id == User.user_id).join(Vehicle, Rental.vehicle_id == Vehicle.vehicle_id).join(Insurance, Rental.policy_number == Insurance.policy_number).all()
         return [rental.serialize() for rental in rentals]
     else:
-        abort(403, description="Invalid permissions")
\ No newline at end of file
+        abort(403, description="Invalid permissions")
+
+@rentals.route('/', methods=['POST'])
+def add():
```

Comment:
Czy tutaj nie trzeba użyć `@login_required`?

Reply:
dodałem (nie wiedziałem czym jest to, a raczej jak sie używa "@login_required" tak wgle XD)